### DISCUSSION TITLE
Welcome to DashboardPlusPlus Discussions!

### DISCUSSION THREAD

**Discussion by TfTHacker** [TfTHacker/DashboardPlusPlus#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jackli89** [comment]
Hi, the snippet for readable line length does not work for me. Any suggestions?

**Reply by TfTHacker** [reply]
Just to review the basics. Do you have it enabled in appearances? Also what theme are you using?

**Reply by jackli89** [reply]
I do have it enabled. I also noticed you added it to the main dashboard snippet file. I'm using the minimal theme.

**Reply by jackli89** [reply]
I figured out the issue. The Contextual Typography plugin was causing some issues with the css code. Not sure why or how but when I disabled it, your snippet was working.

**Reply by jackli89** [reply]
I was using that with the Minimal theme to get image grids just fyi.

**Reply by TfTHacker** [reply]
good to know! Glad it is resolved. Please let me know if I can be of further help.

**Comment by DryIce1** [comment]
Hi @TfTHacker,

thanks for the write up and code! 
I thought it might look nicer with bold in the subtopic, and the remainder of the list in normal font. What do you think?

![image](https://user-images.githubusercontent.com/59501376/160274839-da843eb3-77db-4d37-b26b-c5cdb6e8a152.png)

```
.dashboard div > ul {
    list-style: none;
    display: flex;
    column-gap: 50px;
    row-gap: 30px;
    flex-flow: row wrap;
    font-weight: bold;
}

.dashboard ul {
    font-weight: normal;
}
```

(please note, i have no coding training!)

**Reply by TfTHacker** [reply]
Yeah this is great. I will borrow this from you :-)

**Comment by DryIce1** [comment]
@TfTHacker, I was also interested in adapting it to Live Preview. But I can't get it to work. 

I have tried inserting `is-live-preview`, but the columns do not render.  Have you tried?
Thanks.

**Reply by TfTHacker** [reply]
Yes this won't work in live preview. It has to do with the underlying HTML of live preview. I tried to find a solution, but did not succeed.

**Comment by osharp** [comment]
I recently ran into a problem with Dataview - wanted to enumerate a set of topics that I was working on and put them in the dashboard (choose six notes from the a set marked with a tag). That worked in the past, but recently started putting in an extra space after each line. I found this snippet online and it fixed the problem, so thought I'd share:

`ul.dataview {
    row-gap: 0px !important;
    list-style-type: disc !important;
}
`

The URL where I found it was [here](https://scribehow.com/shared/Using_Dataview_queries_in_Dashboard_pages_in_Obsidian__uTvk--XnTSWtmVGf7zsYUg).

Now I can run this Dataview command, and six of my notes marked with "#processing" all show up as list items in a dashboard section, correctly formatted:
	`$=dv.list(dv.pages('#processing').where(f => !f.file.path.includes("Template") && !f.file.path.includes("System")).limit(6).file.link)`

Thanks for a very cool resource!
O

**Reply by filmgal** [reply]
Thank you osharp!  I wrote about the same issue yesterday: https://github.com/TfTHacker/DashboardPlusPlus/discussions/4#discussion-4143914

Your fix works!  Thanks for posting. It was driving me nuts.

**Comment by GoulartNogueira** [comment]
### Why is this not listed at Community Plugins?

*Or is it? (I'm noob at Obsidian)*

I've just found your dashboard, and I wanted to test it, so I went to Community Plugins.
I found a lot of plugins made by TfTHacker (well done btw!) but I did not find the Dashboard++
Why is that?
Is it due to some technical issue?
Or is this some kind of pre-release version, you intend to publish soon?

**Reply by TfTHacker** [reply]
Hi there. Dashboard++ isn't a plugin, so it won't show up in the plugins. It is a method for creating dashboards. The method is made up of two parts: 1) a CSS Snippet for formatting pages you want to be dashboards in a special way and 2) those pages you want to be dashboards use special markup to make the dashboard.

This repository on GitHub is an Obsidian vault. You can download the vault and open it in Obsidian to learn more. Of course, there is an accompanying article that explains the tools used to make this dashboard system.

Welcome to Obsidian!
